src/main/java/com/google/netpcapanalysis/models/MockPCAP.java

```diff
@@ -0,0 +1,51 @@
+package com.google.netpcapanalysis.models;
+
+import com.google.netpcapanalysis.interfaces.models.PCAP;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class MockPCAP implements PCAP {
+
+  private String id;
+  private List<InetAddress> ips;
+
+  public MockPCAP() {
+    this.id = "sampleId";
+    this.ips = new ArrayList<>();
+    for (int i = 0; i < 1000; i++) {
+      ips.add(genIP());
+    }
+  }
+
+  public String getId() {
+    return id;
+  }
+
+  @Override
+  public List<InetAddress> getIPs() {
+    return ips;
+  }
+
+  public static InetAddress genIP() {
```

Comment:
no reason this should be static. It can be a non-static method.

Reply:
Fixed.